apps/services/user-profile/src/app/v2/user-profile.service.ts

```diff
@@ -82,12 +86,11 @@ export class UserProfileService {
       )
     }
 
-    const { nationalNumber, parsedPhoneNumber } =
+    const formattedPhoneNumber =
```

Comment:
If the phone number is not defined then `formattedPhoneNumber` will be `false` and when islyklar service is called with `false` as phone number it will result in `500 Internal Server Error`.. This should by `const formattedPhoneNumber = isMobilePhoneNumberDefined ? formatPhoneNumber(userProfile.mobilePhoneNumber) : null`